site/_layouts/home.html

```diff
@@ -2,68 +2,77 @@
+                <a class="flex grow-large ml1 ml2-l link-svg link" href={{ app.lighthouse-link }} target="_blank" rel="noopener">
+                    <span class="visuallyhidden">View Lighthouse results</span>
+                    {% include link-icon.svg %}
+                </a>
               </div>
             </dd>
             <dt class="f6 f4-ns b mt2">Interactive (Emerging Markets)</dt>
             <dd class="ml0 f5 fw5 f4-m f3-l pt1">
-              <div class="flex items-center justify-center justify-start-l silver">
+              <div class="flex items-center justify-center justify-start-l mid-gray">
                 {{ app.interactive-em }}
-                <a class="flex grow-large ml1 ml2-l link-svg" href={{ app.wpt-em-link }} target="_blank" rel="noopener">{% include link-icon.svg %}</a>
+                <a class="flex grow-large ml1 ml2-l link-svg link" href={{ app.wpt-em-link }} target="_blank" rel="noopener">
+                    <span class="visuallyhidden">View Interactive (Emerging Markets) results</span>
+                    {% include link-icon.svg %}
+                </a>
               </div>
             </dd>
             <dt class="f6 f4-ns b mt2">Interactive (Faster 3G)</dt>
             <dd class="ml0 f5 fw5 f4-m f3-l pt1">
-              <div class="flex items-center justify-center justify-start-l silver">
+              <div class="flex items-center justify-center justify-start-l mid-gray">
                 {{ app.interactive-faster-3g }}
-                <a class="flex grow-large ml1 ml2-l link-svg" href={{ app.wpt-faster-3g-link }} target="_blank" rel="noopener">{% include link-icon.svg %}</a>
+                <a class="flex grow-large ml1 ml2-l link-svg link" href={{ app.wpt-faster-3g-link }} target="_blank" rel="noopener">
+                    <span class="visuallyhidden">View Interactive (Faster 3G) results</span>
+                    {% include link-icon.svg %}
+                </a>
               </div>
             </dd>
           </dl>
 
           <div class="flex justify-center justify-start-l ">
-            <a class="f7 f6-m fw5 fw6-m tracked bg-orange link dim br-pill dib ph3 ph4-l pv2 white" href={{ app.app-link }}>VIEW APP</a>
-            <a class="f7 f6-m fw5 fw6-m tracked bg-orange link dim br-pill dib ph3 ph4-l pv2 white ml2" href={{ app.github-link }}>SOURCE CODE</a>
+            <a class="f7 f6-m fw5 fw6-m tracked bg-orange link grow br-pill dib ph3 ph4-l pv2 black" href={{ app.app-link }}>VIEW APP</a>
+            <a class="f7 f6-m fw5 fw6-m tracked bg-orange link grow br-pill dib ph3 ph4-l pv2 black ml2" href={{ app.github-link }}>SOURCE CODE</a>
           </div>
         </div>
       </div>
       <div class="fl-l dn db-l">
         <div class="flex h-100 items-center">
-          <a href={{ app.app-link }}>
+          <a class="link" href={{ app.app-link }}>
             <img class="grow w5" src={{ app.image | relative_url }}>
```

Comment:
Need `alt` otherwise it just reads the image filename. Suggest something like `alt=React HN PWA running on a mobile phone`